### ISSUE TITLE
Feature request: USD value of transaction

### ISSUE THREAD

**Issue by zanglebert** [spesmilo/electrum#785]
Hey

I hope this is the right place for my question (and not the electrum subforum on bitcointalk)...

I'm wondering how much effort it would be to calculate the USD (or ideally: any fiat currency) value of a transaction that the user is about to send.

Here's how I imagine it: In the usual tab for sending a transaction, next to the box showing the amount to send and the amount for the fee another box would give the current USD value of those BTC amounts.

Exchange rate data would ideally come from bitcoinaverage.com, in order not to rely on any single exchange.

Why would it be a good idea? For me at least, the greatest hassle, and greatest fear, is sending the "wrong" amount of bitcoins. Say I have a few BTC in my wallet, and now I just want to pay for a pizza online. In those cases, I always have to double and triple check if I don't send 0.5 BTC instead of 0.05 BTC... you get the idea. One decimal place wrong, and I send out an order of magnitude higher value, with no way to reverse the transaction (which is good, in principle :D)

In other words: As much as we try to "think in Bitcoins", the more familiar unit of account so far is USD, and I think it would be extremely useful to see the USD value next to the BTC value.

Any chance this can be implemented without too much effort? I don't know how difficult it would be to bind in the external data from, say, bitcoinaverage.com. Could that be done through the existing server functionality, or could electrum get the exchange data directly? The rate is ultimately not really of any sensitive nature, so a centralized point of getting it shouldn't be a problem, right?
